server/pbench/bin/pbench-move-unpacked

```diff
@@ -44,15 +44,16 @@ linkdest=MOVED-UNPACKED
 echo $TS
 
 # get the list of files we'll be operating on - sort them by size
-list=$UNPACK_PATH/pbench-unpack-tarballs.$$
 tmp=$TMP/$PROG.$$
+list=$tmp/$PROG.$$.list
```

Comment:
Now that you moved `$list` inside $tmp (which is already named with $$), you don't need the second $$.